source/platform/native.rst

```diff
@@ -4,7 +4,7 @@
 Native Support
 ==============
 
-As of this writing, Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ support Adventure
+As of this writing, Velocity 1.1.0+, Sponge 8+, Paper 1.16.5+ and Minestom support Adventure
```

Comment:
Should this include an oxford comma?

Reply:
Yes

Reply:
Just letting my hatred of Oxford commas get the better of me again...